documentation/specs/src/interoperability/ethereum-bridge/bootstrapping.md

```diff
@@ -1,4 +1,73 @@
 # Bootstrapping the bridge
 
-Before the bridge can start running, some storage may need to be initialized in 
-Namada. TBD.
\ No newline at end of file
+## Overview
+
+The Ethereum bridge is not enabled at the launch of a Namada chain. Instead, there is a governance parameter, `eth_bridge_proxy_address`, which is initialized to the zero Ethereum address (`"0x0000000000000000000000000000000000000000"`). An overview of the steps to enable the Ethereum bridge for a given Namada chain are:
+
+- A governance proposal should be held to agree on a block height `h` at which to launch the Ethereum bridge by means of a hard fork.
+- If the proposal passes, the Namada chain must halt after finalizing block `h-1`. This requires
+- The [Ethereum bridge smart contracts](./ethereum_smart_contracts.md) are deployed to the relevant EVM chain, with the active validator set at block height `h` as the initial validator set that controls the bridge.
```

Comment:
I'm not familiar with providing an initial nonce when deploying the smart contract, if this is just a semi-arbitrary number (e.g. epoch, block height), it could be agreed during the governance proposal, if it needs to be signed as well by the validator set then we'll need some extra steps

Reply:
I think we reverted back using the `version` field. Since upgrading the smart contract requires an hardfork, we can have `version` in the genesis file (and then added to the eth bridge parameter storage)